CHANGELOG.md

```diff
@@ -15,7 +15,7 @@
 
 ### 0.9.17
 
-* The `lobster-python` tool now adds the line number to the function
+* The `lobster-python` tool now adds the counter logic to the function
```

Comment:
Don't modify the change log of version `0.9.17`. That version is already published. We cannot change it any more. Please describe the recent changes under the section regarding version `0.9.18`.